app/src/main/res/values-v31/colors.xml

```diff
@@ -6,23 +6,20 @@
 -->
 
 <resources xmlns:android="http://schemas.android.com/apk/res/android">
-    <!-- System theming colors for LXX_Light theme, overriding some colors in main colors.xml -->
-    <color name="highlight_color_lxx_light">@android:color/system_accent1_500</color>
-    <color name="gesture_trail_color_lxx_light">@android:color/system_accent1_200</color>
-    <color name="sliding_key_input_preview_color_lxx_light">@android:color/system_accent1_900</color>
-    <color name="action_key_background_lxx_light">@color/highlight_color_lxx_light</color>
-    <color name="action_key_background_pressed_lxx_light">@android:color/system_accent1_200</color>
-    <color name="icon_tint_system_accent_lxx_light">@color/highlight_color_lxx_light</color>
+    <!-- Colors for Dynamic colors light theme -->
+    <color name="action_key_background_dynamic_light">@android:color/system_accent1_200</color>
```

Comment:
Why do you use this instead of directly getting the colors like `@android:color/system_accent1_200`?
It seems to me that directly using those colors would show more clearly that system colors are used.

Reply:
Sorry again but I don't understand your comment...

Reply:
`action_key_background_dynamic_light` always has the value `@android:color/system_accent1_200`, so you could just use `android.R.color.system_accent1_200` in `KeyboardTheme`

Reply:
I didn't do it to remain consistent with what was already coded. It seemed logical to me to have values in the colors.xml file like the majority of colors indicated in _KeyboardTheme.getThemeColors_.
I'll apply what you tell me.

However, I have a question:
For example: `key_text_color_lxx_dark` will also always have the same color; so why not write `Color.parseColor(#FFFFFF)` instead of `ContextCompat.getColor(context, R.color.key_text_color_lxx_dark)` ?

Reply:
Solved with commit 35c4da4

Reply:
> For example: key_text_color_lxx_dark will also always have the same color; so why not write Color.parseColor(#FFFFFF) or Color.rgb(255, 255, 255) instead of ContextCompat.getColor(context, R.color.key_text_color_lxx_dark)

The colors were already defined in resources and used for exactly that. When overhauling the theming system it was faster to just take the resource values than look them up.
The whole color mess in resouces is still something on my clean up todo list, but very low priority.

Reply:
Technically `android.R.color.system_accent1_200` is still in resources, but coming Android instead of app (but whatever, that's not the point)

Reply:
Okay, that's what I thought. 😉